packages/core/src/components/DismissableBanner/DismissableBanner.tsx

```diff
@@ -58,9 +67,15 @@ type Props = {
   variant: 'info' | 'error';
   message: ReactNode;
   id: string;
+  fixed?: boolean;
 };
 
-export const DismissableBanner: FC<Props> = ({ variant, message, id }) => {
+export const DismissableBanner: FC<Props> = ({
```

Comment:
Just a side note since it's not part of your changes, but we [avoid FC](https://github.com/spotify/backstage/blob/master/docs/architecture-decisions/adr006-avoid-react-fc.md). You don't need to change anything here, only informational :)